apps/mobile/src/libs/helpers/get-fonts-family/get-fonts-family.ts

```diff
@@ -0,0 +1,12 @@
+import { fontFamilies } from "~/libs/enums/fonts/fonts-families.enum";
+
+const getFontFamily = (
```

Comment:
Do we really need this helper? Why can't we just pass fontFamily and fontWeight to our custom Text component?

Reply:
Delete getFontFamily.